### PR TITLE
Add modernization section to poem how-to

### PR DESCRIPTION
**Pull request by EmmaSweeney**

### CODE REVIEW

File: www/contribute/how-tos/how-to-structure-and-style-large-poetic-productions.php [hunk 1]
```diff
@@ -17,6 +17,20 @@
 					<li><p><a href="#a-single-collections-file">A single collections file</a></p></li>
 					<li><p><a href="#multiple-files">Multiple files</a></p></li>
 				</ol>
+				<li><p><a href="#modernizing-poetic-works">Modernizing poetic works</a></p></li>
+				<ol>
+					<li><p><a href="#modernize-prose-frontmatter-and-backmatter">Modernize prose frontmatter and backmatter</a></p></li>
+					<li><p><a href="#modernize-poems-carefully">Modernize poems <i>carefully</i>!</a></p></li>
+					<li><p><a href="#modernization-exceptions">Modernization exceptions</a></p></li>
+					<ol>
+						<li><p><a href="hyphenations-and-spacings">Hyphenations and spacings</a></p></li>
```
Comment (acabal) [inline www/contribute/how-tos/how-to-structure-and-style-large-poetic-productions.php:26]: I think all of these are missing `#` hashes?

**Review by acabal — CHANGES_REQUESTED** [review]
Great! Small note in the ToC.

File: www/contribute/how-tos/how-to-structure-and-style-large-poetic-productions.php (hunk 1 above)
Comment (EmmaSweeney) [reply www/contribute/how-tos/how-to-structure-and-style-large-poetic-productions.php]: These have been fixed.

**Comment by acabal** [comment]
Great work, thanks!

**Pull request merged**